pkgs/development/tools/metals/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, lib, coursier, jdk, jre, python, makeWrapper }:
```

Comment:
`python` doesn't appear to be used. Is it meant to be here?

Reply:
No it's an unfortunate copy/paste. I'll remove it.